dotcom-rendering/scripts/webpack/browser.js

```diff
@@ -59,6 +60,13 @@ module.exports = ({ isLegacyJS }) => ({
 	plugins: [
 		DEV && new webpack.HotModuleReplacementPlugin(),
 		DEV && friendlyErrorsWebpackPlugin(),
+		DEV &&
+			new GuStatsReportPlugin({
+				buildName: isLegacyJS ? 'client' : 'legacy-client',
```

Comment:
Hello @OllysCoding sorry for chiming in late. Is this ternary the right way around?